src/status_im/data_store/realm/schemas/account/contact.cljs

```diff
@@ -25,3 +25,30 @@
                                          :optional true}
                       :debug?           {:type    :bool
                                          :default false}}})
+
+(def v2 {:name       :contact
+         :primaryKey :whisper-identity
+         :properties {:address          {:type :string :optional true}
+                      :whisper-identity :string
+                      :name             {:type :string :optional true}
+                      :photo-path       {:type :string :optional true}
+                      :last-updated     {:type :int :default 0}
+                      :last-online      {:type :int :default 0}
+                      :pending?         {:type :bool :default false}
+                      :hide-contact?    {:type :bool :default false}
```

Comment:
I'm not 100% sure, but this is probably not needed anymore and was relevant when we had "bot" contacts, seems like a good opportunity to get rid of it.